docs/organizations/verified-domains.mdx

```diff
@@ -1,38 +1,57 @@
+Enabling verified domains applies to all organizations and cannot currently be managed on a per-organization basis.
 
-You can create up to 10 domains per organization to meet your needs. If you need more than 10 domains, reach out to support@clerk.dev.
+In order to enable this feature:
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations-settings)
+2. In the navigation sidebar, select **Organizations Settings**.
+3. In the **Verified domains** section, check the **Enable verified domains** checkbox.
+4. The following settings will appear:
+  - [**Enrollment mode**](#enrollment-mode) - Choose between `automatic_invitation` and `automatic_suggestion`.
+  - [**Default role**](#default-role) - Choose the default role for users who join the organization through a verified domain.
+
+### Enrollment mode
+
+You can enable the following enrollment modes to be available for your application:
+
+- `automatic_invitation` - During sign-up, a user will receive an invitation for the organization if their email's domain matches the verified domain. The user will join the organization if they accept the automatic invitation.
+- `automatic_suggestion` - During sign-up, a user will receive a suggestion for the organization if their email's domain matches the verified domain. The user can request to join, and an administrator must accept this request before the user can join the organization.
 
-## Default role
+Once a domain is created and verified in your application, an administrator will have the option to enable whichever enrollment modes you made available.
 
-When enabling the verified domains feature, you are also required to define which role will be assigned to the user that will receive the organization invitation or organization suggestions in order to join your organization.
+### Default role
 
-You can choose any of the available organization roles of your instance.
+Once verified domains are enabled, you are required to define which role will be assigned to the user that will receive the organization invitation or organization suggestions in order to join your organization.
+
+By default, the options are "Member" and "Admin". However, you can also [create custom roles for your organization](/docs/organizations/create-roles-permissions).
 
 <Callout type="warning">
   You can't delete an organization role if it's used as the verified domain's default role.
 </Callout>
 
-To specify the verified domain's default role, in the Clerk Dashboard, go to the **[Organizations Settings](https://dashboard.clerk.com/last-active?path=organizations-settings)** page.
+## Adding and verifying domains
 
-## Enrollment modes
+Domains can be added and verified under an organization by an organization administrator. The easiest way to do so is by using Clerk's [`<OrganizationSwitcher />`] component. In the **General** tab, there will be a **Verified domains** section where you can add and verify domains.
```

Comment:
This is not restricted to organization admins. Any user with the proper permission `org:sys_domains:manage` can do it